examples/CIBFE/ex0/example.cpp

```diff
@@ -344,7 +344,7 @@ run_example(int argc, char* argv[])
         {
             time_integrator->registerVisItDataWriter(visit_data_writer);
         }
-        libMesh::UniquePtr<ExodusII_IO> exodus_io(uses_exodus ? new ExodusII_IO(mesh) : NULL);
+        std::unique_ptr<ExodusII_IO> exodus_io(uses_exodus ? new ExodusII_IO(mesh) : NULL);
```

Comment:
NULL ---> nullptr?

Reply:
...oh wait, you did not run modernize on the examples, did you?

Reply:
The examples have not been updated to C++11 constructs except for std::unique_ptr.